be/src/olap/rowset/segment_v2/segment_iterator.cpp

```diff
@@ -938,6 +938,10 @@ Status SegmentIterator::_apply_inverted_index_on_block_column_predicate(
 }
 
 bool SegmentIterator::_need_read_data(ColumnId cid) {
+    // for safety reason, only support DUP_KEYS
+    if (_opts.tablet_schema->keys_type() != KeysType::DUP_KEYS) {
```

Comment:
consider unique table with MOW

Reply:
not support MoW now, because MoW is not always generate delete bitmap for delete predicate